templates/github/.github/workflows/scripts/install.sh.j2

```diff
@@ -52,6 +58,34 @@ if [[ -f ../../ci_requirements.txt ]]; then
     ci_requirements: true
 VARSYAML
 fi
+{%- if test_s3 %}
+if [ "$TEST" = "s3" ]; then
+  cat >> vars/main.yaml << VARSYAML
+    pulpcore: "${PULPCORE_PREFIX}[s3]"
```

Comment:
I refactored this bit so that the generated `install.sh` is easier to read, see https://github.com/pulp/pulp_python/pull/638/files#diff-02059c243ab1ae5faa6a95ec965f0b351e0f6c96e5452c90881b6b7e3ee11618R40-R46